pkg/controller/controller_test.go

```diff
@@ -457,3 +457,81 @@ func TestGetEmail(t *testing.T) {
 		})
 	}
 }
+
+func TestGetCatagoriesDisplayData(t *testing.T) {
+	t.Parallel()
+
+	validators := map[string]jvspb.Validator{
+		"jira": &mockValidator{
+			Valid:       true,
+			DisplayName: "Jira issue key",
+			Hint:        "Jira Issue key under JVS project",
+		},
+		"git": &mockValidator{
+			Valid:       false,
+			DisplayName: "Git issue key",
+			Hint:        "Git Issue key under JVS project",
+		},
+	}
+	jiraUIData := &jvspb.UIData{
+		DisplayName: "Jira issue key",
+		Hint:        "Jira Issue key under JVS project",
+	}
+	gitUIData := &jvspb.UIData{
+		DisplayName: "Git issue key",
+		Hint:        "Git Issue key under JVS project",
+	}
+	expectedCatagoriesData := map[string]*jvspb.UIData{
+		"jira": jiraUIData,
+		"git":  gitUIData,
+	}
```

Comment:
Inline all these in line 503.

Reply:
Moved all these into the second test case